core/src/main/javascript/reportviewer/report.html

```diff
@@ -36,7 +36,10 @@
       disableFlashStorage: true
     };
 
-
+    var inSchedulerDialog = window.location.pathname.toLowerCase().indexOf('parameterui') >= 0;
```

Comment:
The existing responsive parameter UI contract involves two parties.

One in the schedule parameters dialog, the host. It has the following styles applied:
https://github.com/pentaho/pentaho-commons-gwt-modules/blob/3d744604b2d1c211ce84faa341f59b074ef0e87c/widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css#L2064-L2083

The other party is the HTML document of the parameter UI for a certain type of scheduled content, which is displayed in an iframe in the host party. A parameter UI document that opts in to responsive mode (for responsive themes) must specify the classes `responsive` and `parameter-ui` on its `body` element. The following styles will apply, on responsive themes (atm, only the ruby theme):
https://github.com/pentaho/pentaho-commons-gwt-modules/blob/3d744604b2d1c211ce84faa341f59b074ef0e87c/widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css#L1942C5-L1946

The change being done here likely does not achieve the same level of responsiveness, given the host `iframe` will always have the same size, and, ideally, there should be a single vertical bar displayed on the host document, and not inside the `iframe`. Also, this change affect all themes.

This change is acceptable, compared to using the "responsive parameter UI contract" presented above, given that the contained prompt parameter components, and the prompt`s overall layout, are not prepared to behave responsively, yet. And that will take more work.